spouse/tf_model.py

```diff
@@ -2,56 +2,68 @@
 from typing import Tuple
 import numpy as np
 import pandas as pd
+
 import tensorflow as tf
+from tensorflow.keras.layers import (
+    Bidirectional,
+    Concatenate,
+    Dense,
+    Embedding,
+    Input,
+    LSTM,
+)
 
 
 def get_feature_arrays(df: pd.DataFrame) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
     """Get np arrays of upto max_length tokens and person idxs."""
+    bet = df.between_tokens
+    left = df.apply(lambda c: c.tokens[: c.person1_word_idx[0]][-4:-1], axis=1)
```

Comment:
this would only pick up the last 3 tokens, not all of them right?

Reply:
Yes, that is intentional (the existing right_tokens field also does that).